### DISCUSSION TITLE
Publishing ESM for Node

### DISCUSSION THREAD

**Discussion by wolfy1339** [octokit/.github#68] [Ideas]
Since work on `octokit-next` is stalled, I suggest that we publish an ESM build that is simply 1:1 of our TypeScript source code.
This should help unblock many of our dependency upgrades.

Once work resumes on `octokit-next`, we can use that instead of the current builds.

We could also publish a CJS version alongside for compability

**Comment by gr2m** [comment]
I like the idea, my recommendation would be not to try to ship both an ESM and CJS version, I'd make a clean cut and go all in on ESM, I have constantly problems with projects that try to do both
